test/load/paypalcontroller.js

```diff
@@ -0,0 +1,54 @@
+import http from "k6/http";
+import { check, fail } from "k6";
+
+const REGIONS = __ENV.REGIONS.split(",") || ["US", "EU"];
+const BILLING_RELAY_URL = __ENV.BILLING_RELAY_URL;
+const BILLING_PAYPAL_WEBHOOK_KEY = __ENV.BILLING_PAYPAL_WEBHOOK_KEY;
+
+export const options = {
+  scenarios: {
+    single_user: {
+      executor: "constant-vus",
+      vus: 1,
+      duration: "30s",
+      tags: { test_type: "single_user" },
+    },
+  },
```

Comment:
🎨 This can get a lot more complicated of course, and some of my `server` load tests show that. This could be helpful as an input parameter, albeit complicated to send JSON through.

Reply:
I wanted to start with something simple for now, then come back with a [multi-scenario](https://grafana.com/docs/k6/latest/using-k6/scenarios/) approach once I see this in action for this repo.